content/en/docs/Installation/nvidia_agx_orin.md

```diff
@@ -181,12 +173,31 @@ If you are editing the partition sizes and generating the images manually, use t
         </partition>
 ```
 
+Be mindful also to change the esp partition or add it if required:
+
+```xml
+     <partition name="esp" type="data">
+            <allocation_policy> sequential </allocation_policy>
+            <filesystem_type> basic </filesystem_type>
+	    <size> 20971520 </size>
+	    <file_system_attribute> 0 </file_system_attribute>
+	    <partition_type_guid> C12A7328-F81F-11D2-BA4B-00A0C93EC93B </partition_type_guid>
+	    <allocation_attribute> 0x8 </allocation_attribute>
+	    <percent_reserved> 0 </percent_reserved>
+            <filename> efi.img </filename>
+            <description> **Required.** Contains a redundant copy of CBoot. </description>
+        </partition>
+```
+
+You can also remove the other partitions under `sdmmc_user` as not effectively used by Kairos during boot.
+{{% /alert %}}
+
 {{% alert title="Note" %}}
 The `COS_PERSISTENT` partition is optional. You can also use an SD card, or an nvme drive instead. The only requirement is to have the partition labeled as `COS_PERSISTENT`.
 {{% /alert %}}
 
 {{% alert title="Note" %}}
-If modifiying the parition sizes, you need to replace `<size>` of each partition in the XML:
+If modifiying the parition sizes, you need to replace the size inside the `<size></size>` tags of each partition in the XML:
```

Comment:
Do we mention anywhere what the `size` is measured in? (e.g. Mb). Same thing for our settings below (e.g. `STATE_SIZE`)

Reply:
nope, but we provide below exact commands to get that value to replace in here (`stat..`)